tmail-backend/combined-identity/src/test/resources/ldif-files/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM dinkel/openldap:latest
```

Comment:
I think it will not work.
The latest JAMES SHA is based on another container (another configuration in Java class `org.apache.james.user.ldap.LdapGenericContainer.Builder#createContainer`)

Reply:
@vttranlina I just reverted your all upgrade James PR. I see all work based on master failing their builds related to those changes so I think reverting it for now is the best way to go